db/seeds.rb

```diff
@@ -0,0 +1,44 @@
+Car.create!(
```

Comment:
- [ ] This seeds file does not work for new users:

![image](https://github.com/HERMON-1995/Book_vehicle_rental-backend/assets/57421823/0468690b-3286-426b-9d42-2f6fda3f861a)

This is because it creates a lot of cars, all of which depend on a user id:

![image](https://github.com/HERMON-1995/Book_vehicle_rental-backend/assets/57421823/eaacd10e-4b5a-4a36-9571-39315db80fad)

This means there must be at least a user existing before this file can be run.

Kindly, add a user creation at the top of this file, before the Car creation statements and it will work. 👍 

This is an example:

![image](https://github.com/HERMON-1995/Book_vehicle_rental-backend/assets/57421823/bd35d008-eea6-4c69-8e21-102d9fa4627a)

Reply:
User seed added.